### DISCUSSION TITLE
can't find files in global folder

### DISCUSSION THREAD

**Discussion by not-tia** [alefragnani/vscode-language-pascal#131] [Q&A] [answered]
I was following the steps to install global and gtags and all that, but when trying to add them to PATH i realized that there isn't a `bin` folder inside the `global-6.6.9` folder (also it isn't called `glo653wb` like in the readme) and also that there isn't a `ctags.conf` file (also no folders called ctags58 in the ctags folder). Any help is appreciated, thanks.

**Comment by alefragnani** [comment, marked as answer]
Hi @Esmack738 ,

It's been a long time I don't download both `global` or `ctags`, but I just downloaded **global 6.6.9** (the recent one) and not only the `bin` folder does exists but also the `share\gtags\gtags.conf` file. The same for **ctags 5.8**.

BTW, the `glo653wb` folder in readme refers to global v6.5.3, that I've been using since I developed the extension. And `ctags58` refers to ctags v5.8, also the same one I've been using since the beginning, and coincidentally is still the latest version available.

Be sure you are downloading the _binaries_, not the sources.

Hope this helps

**Reply by BTD684** [reply]
uhm... where can I download the ver with **bin**? i got mine from https://ftp.gnu.org/pub/gnu/global/ 
experiencing the same problem as the person above, can't find **bin**

**Reply by alefragnani** [reply]
Hi @BTD684 ,

Based on the ftp url you provided, I guess you are downloading the _sources_. If that´s the case,  you need to need to _build_ the binaries.  The download pages seems to provide the steps, but I never did it, so I can´t say how easy it is. 

I only used Windows versions, so the download already comes with the **bin** folder.

Hope this helps
